etc/adminhtml/system/account.xml

```diff
@@ -311,6 +311,15 @@
                 </depends>
             </field>
 
```

Comment:
scope is global, therefore show in website and store view - should not be present